<launch>

  <node pkg="prophesee_ros_driver" type="prophesee_ros_publisher" name="prophesee_ros_publisher" respawn="false" output="screen">
    <param name="camera_name" value="camera" />
    <param name="publish_cd" value="true" />

    <!-- Path to the file with the camera settings -->
    <param name="bias_file" value="/home/yuxuan/Documents/Prophesee/out.bias"/>

    <!-- Instead of live camera, you can read data offline from raw file -->
    <param name="raw_file_to_read" value="" />

    <param name="trigger_reconstruct" value="true" />
    <param name="trigger_separate_usec" value="5000" />

    <!-- approximate time span in seconds for which events will
	 be aggregated until a ROS message is generated. Defaults to
	 100 usec -->
    <param name="event_delta_t" value="0.000100" />
  </node>

</launch>
